apps/src/lib/levelbuilder/unit-editor/CloneLessonDialog.jsx

```diff
@@ -103,6 +116,33 @@ export default class CloneLessonDialog extends Component {
                 onChange={e => this.setState({destinationUnit: e.target.value})}
               />
             </label>
+            <label>
+              Level copy type:{' '}
+              <select
+                className="levelCopySelector"
+                value={this.state.levelCopyType}
+                style={styles.dropdown}
+                onChange={e => this.setState({levelCopyType: e.target.value})}
+              >
+                {Object.keys(LevelCopyText).map(type => (
+                  <option key={type} value={type}>
+                    {LevelCopyText[type]}
+                  </option>
+                ))}
+              </select>
+            </label>
+            {this.state.levelCopyType === 'deepCopy' && (
+              <label>
+                Suffix to add when copying levels:{' '}
+                <input
+                  type="text"
+                  value={this.state.newLevelSuffix}
+                  onChange={e =>
```

Comment:
If you set this and then pick shallow copy does this value clear?

Reply:
It looks like you get around this with line 38/39 so I guess it doesn't need to clear.

Reply:
right, I'm not actually clearing the value of the new level suffix in the react state, then I'm using a unit test to make sure we still send the right data to the server.